backend/pkg/database/scheduler.go

```diff
@@ -72,6 +74,8 @@ func (s *SchedulerTableHandler) Set(scheduler *Scheduler) error {
 		return fmt.Errorf("failed to set scheduler: %v", err)
 	}
 
+	refreshMaterializedViewsIfNeeded(s.db, s.driverType)
+
 	return nil
```

Comment:
Data is updated in the "UpdateNextScanTime" function below should we have a view refresh there too?

Reply:
we might still need a s.setViewRefreshHandlerIfExists() in the UpdateNextScanTime below